src/gen/conditions.rs

```diff
@@ -2778,6 +2943,50 @@ fn test_agg_sig_unsafe_extra_arg() {
     );
 }
 
+#[cfg(test)]
+#[rstest]
+#[case(AGG_SIG_PARENT)]
+#[case(AGG_SIG_PUZZLE)]
+#[case(AGG_SIG_AMOUNT)]
+#[case(AGG_SIG_PUZZLE_AMOUNT)]
+#[case(AGG_SIG_PARENT_PUZZLE)]
+#[case(AGG_SIG_PARENT_AMOUNT)]
+fn test_agg_sig_ph_extra_arg(#[case] condition: ConditionOpcode) {
```

Comment:
We're missing a case for `AGG_SIG_ME` here. Also `ph` in the name would become less informative now that we're covering several cases.

Reply:
I addressed this by removing the `AGG_SIG_ARGS` flag, and renamed the test function